mesheryctl/pkg/utils/sse-client.go

```diff
@@ -88,23 +89,11 @@ func loop(reader *bufio.Reader, events chan Event) {
 
 		default:
 			close(events)
+			return
```

Comment:
here too.
// @DelusionalOptimist

Reply:
Yup this is fine, because at the two places that depend on this function ([here](https://github.com/krithikvaidya/meshery/blob/feature/krithikvaidya/sse_client_tests/mesheryctl/internal/cli/root/mesh/validate.go#L118) and [here](https://github.com/krithikvaidya/meshery/blob/feature/krithikvaidya/sse_client_tests/mesheryctl/internal/cli/root/mesh/deploy.go#L152)), we are using a range loop over the channel that is returned. Once close() is called on the channel, the range loop will terminate on receiving this event (which implies that all events sent over the channel before the close() have also been received), and no more events will be read from the ```events``` channel.

Reply:
CITY!